pacman.js

```diff
@@ -0,0 +1,44 @@
+function Pacman() {
+  this.livesLeft = 2;
+  this.points = 0;
+  this.powerTimeLeft = 0;
+}
+
+Pacman.prototype = {
+  eat: function(food) {
+    if (food instanceof Ball) {
+      this.eatBall(food);
+    } else if (food instanceof Ghost) {
+      this.eatGhost(food);
+    } else {
+      console.log("Food error");
+    }
+  },
+
+  eatGhost: function(ghost) {
+    // console.log("Encountered ghost"); // test message
+    return (this.powerTimeLeft > 0) ? this.points += 1000 : this.die();
+  },
+
+  eatBall: function(ball) {
+    this.powerTimeLeft -= 1;
+    if (ball.type === "super") {
```

Comment:
can we change this to `ball.ballType` please?

Otherwise, great pull request! :+1: